### ISSUE TITLE
Untraceable 500 Error on Linux Server

### ISSUE THREAD

**Issue by hbhargava7** [castlamp/zenbership#187]
I have an AWS EC2 Ubuntu server instance, with prereqs installed as per [this page](http://documentation.zenbership.com/Basics/Installation-and-Setup). Apache2 and PHP both work, as tested by creating a simple test index.php file. OS version is Ubuntu 16.04.4 LTS,

I extracted the Zenbership files to `/var/www/html` on the server, and updated permissions settings.

However, I consistently face a 500 Internal Server Error upon navigating to the server's public address. Initially, `/var/log/apache2/error.log` showed fatal PHP `require()` errors owing to the yet-unconfigured `config.php`, but after updating the paths in that file, no errors are logged in `error.log`, but the 500 error persists.

**Comment by BlueHulk1** [comment]
Hello,
I am also having trouble with zenbership:
Here's the details:
ok - My hosting service is GoDaddy
ok - Read and followed you Installtion instructions
ok - Used Filezilla with "transfer type" set to auto
ok - Changed file permissions to 777 for the required directories
admin/sd-system
admin/sd-system/attachments
admin/sd-system/exports
custom/sessions
custom/qrcodes
custom/uploads

ok - Succesfully created a blank mysql database
ok - Run http://www.tannerstoys.com/gtfree/lifepath/mbr/setup/
ok - Enter Database name, username, password = Success!
ok - I filled out the rest of the form and click on Process Setup
fail -  get the following page error

 http://www.tannerstoys.com/gtfree/lifepath/mbr/setup/process.php
This page isn’t working
 www.tannerstoys.com is currently unable to handle this request.
HTTP ERROR 500

This error occurs inside   mbr/setup/process.php
roughly at line 107, which is the following line:    require "../admin/cp-classes/db.class.php";
That file is minified

**Comment by steviobee** [comment]
Try uploading the zip file to the server, unzip and run from there.
Worked for me but be warned, this isn't ready for a live site as it needs a lot of work still in development

**Comment by BlueHulk1** [comment]
Is there an older version (v114, 113, 112) that is working well enough for me to do some general testing?
Where can I go to download previous versions?

**Comment by Redacted99** [comment]
Maybe echo getcwd() before 107 to see if the require path makes sense?

**Comment by BlueHulk1** [comment]
Yes, that's how I was able to target the problem to that line number.  
---------- Original Message ----------
